homeassistant/components/upnp/strings.json

```diff
@@ -21,7 +21,8 @@
     "step": {
       "init": {
         "data": {
-          "scan_interval": "Update interval (seconds, minimal 30)"
+          "scan_interval": "Update interval (seconds, minimal 30)",
+          "force_poll": "Force polling of all data"
```

Comment:
Could you maybe update the description of the PR to explain what this feature does and how it affects the integration?

Thanks! 👍 

../Frenck

Reply:
Currently the component regularly polls the IGD (router) by calling several so called Actions (Action in UPnP terms, an RPC), such as `GetTotalBytesReceived`. In this case, these Actions simply return a State Variable (another UPnP term, speaks for itself I guess.)

UPnP also specifies that State Variables can be evented. This means that a client can subscribe to a Service (which holds Actions and State Variables) and gets callbacks to when a State Variable changes. This prevents regularly calling Actions, saving some processing on the client side (in this case, the Home Assistant side.)

Please do note that not all State Variables are all evented. The example of Action `GetTotalBytesReceived`, which returns State Variable `TotalBytesReceived` is not evented. The State Variable(s) we are interested in are `ExternalIpAddress` ,`ConnectionStatus`, and `PortMappingNumberOfEntries`.

However, some UPnP devices have a UPnP buggy implementation. Hence, we want the users to be able to fall back to polling for the mentioned State Variables.